File: nutron-auth/src/main/java/com/lazysyntax/nutron/auth/model/dto/UserSetupResponse.java
package com.lazysyntax.nutron.auth.model.dto;

import lombok.Builder;

@Builder
public record UserSetupResponse(
        String weight,
        String height,
        String gender,
        String age,
        String activity,
        String goal,
        String formula,
        String diet
) {

}

File: nutron-app-kmp/composeApp/src/commonMain/kotlin/com/lazysyntax/nutron/data/remote/synchronization/UserSetupResponse.kt
package com.lazysyntax.nutron.data.remote.synchronization

import kotlinx.serialization.Serializable

@Serializable
data class UserSetupResponse(
    val weight: String = "",
    val height: String = "",
    val age: String = "",
    val gender: String = "",
    val activity: String = "",
    val goal: String = "",
    val formula: String = "",
    val diet: String = ""
)